web/app/controller/Root.js

```diff
@@ -25,6 +25,10 @@ Ext.define('Traccar.controller.Root', {
         'Traccar.model.Position'
     ],
 
+    init: function () {
+        Ext.state.Manager.setProvider(new Ext.state.CookieProvider());
```

Comment:
Looks like we can http://docs.sencha.com/extjs/6.2.0/classic/Ext.button.Button.html#cfg-stateful